extensions/data-workspace/package.json

```diff
@@ -5,7 +5,7 @@
   "version": "0.6.0",
   "publisher": "Microsoft",
   "preview": true,
-  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt",
+	"license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/extensions/data-workspace/LICENSE.txt",
```

Comment:
This should be `SEE LICENSE IN LICENSE.TXT` per the docs...

Reply:
updated! but I guess it is not a mandatory one.

Whether i provide the license attribute or not, the outcome still the same. In either scenario, i can see the manifest file with license value picked from the extension/license.txt.

Reply:
My earlier question, do we need to vbump the extensions? if not publishing this will replace the existing available version?

Reply:
In order for this to show up in the gallery - yes you would. But unless there's a requirement (for compliance or something) then I don't think pushing an update just to fix the license link is worth it. Just let it be picked up the next time these are updated with actual functional changes.

Reply:
@chlafreniere / @dzsquared, what is your opinion on this? The support team aims to resolve this issue by today and has tagged it as a policy compliance matter.

Additionally, we do not frequently update these extensions unless there are critical changes required for releases.

Reply:
@ssreerama If the version hasn't been bumped already since the last release, let's vbump. Shipping cadence-wise, plan is to ship with the upcoming April release later this month.

Reply:
@chlafreniere What about the sql bindings extension? We're very unlikely to be bumping that anytime soon, so I guess that should be updated along with the others if this is something we're getting flagged for.

Reply:
In that case will bump all three extension versions!